pkg/validations/createvalidations/gitops.go

```diff
@@ -5,12 +5,25 @@ import (
 	"fmt"
 
 	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/git/providers/git"
 	"github.com/aws/eks-anywhere/pkg/logger"
 	"github.com/aws/eks-anywhere/pkg/types"
 	"github.com/aws/eks-anywhere/pkg/validations"
 )
 
+const (
+	EksaGitPasswordTokenEnv   = "EKSA_GIT_PASSWORD"
+	EksaGitPrivateKeyTokenEnv = "EKSA_GIT_PRIVATE_KEY"
```

Comment:
We have this constant defined at least 3 times, could move it to just one place?

Also, this makes me realize we are reading this env var at least 3 times. Ideally we should read it once and pass it around.

Reply:
So we read the private key file once when we mount the file on the container, we also read both those vars when we  do preflight validations. with the validations, we only return errors throughout which is why there isn't really a way currently to pass these values back.